### PR TITLE
Hotfix

### PR DESCRIPTION
**Pull request by 0rdinal**
Small changes to the legacy code to assist with debugging and remove the config file on the serverside

### CODE REVIEW

**Comment by 0rdinal** [comment]
Also added a commandline argument to set a specific port

**Comment by cron1155** [comment]
It looks good. But did you guys tried to compile it ?

**Comment by 0rdinal** [comment]
Built fine with the changes made here and the dependencies being installed

**Review by cron1155 — APPROVED** [review]

**Comment by 0rdinal** [comment]
@blipvert Would you mind reviewing?

**Review by cron1155 — APPROVED** [review]

**Review by blipvert — APPROVED** [review]

**Review by benkyd — APPROVED** [review]
Don't have time to test but the code looks good

File: legacy/f4mp/f4mp.vcxproj [hunk 1]
```diff
@@ -48,7 +48,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'" Label="Configuration">
     <ConfigurationType>DynamicLibrary</ConfigurationType>
     <UseDebugLibraries>false</UseDebugLibraries>
-    <PlatformToolset>v140</PlatformToolset>
+    <PlatformToolset>v142</PlatformToolset>
```
Comment (Genetical) [inline legacy/f4mp/f4mp.vcxproj:51]: `v142` not compatible with VS2015
```suggestion
    <PlatformToolset>v140</PlatformToolset>
```

File: legacy/f4mp_server/f4mp_server.vcxproj [hunk 2]
```diff
@@ -41,7 +41,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <UseDebugLibraries>true</UseDebugLibraries>
-    <PlatformToolset>v142</PlatformToolset>
+    <PlatformToolset>v140</PlatformToolset>
```
Comment (Genetical) [inline legacy/f4mp_server/f4mp_server.vcxproj:44]: `v142` not compatible with VS2015
```suggestion
    <PlatformToolset>v140</PlatformToolset>
```

**Pull request merged**